modules/passkey/contracts/SafeWebAuthnSignerProxyHarness.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+/* solhint-disable no-complex-fallback */
+pragma solidity >=0.8.0;
+
+import {P256} from "./libraries/WebAuthn.sol";
+import {SafeWebAuthnSignerProxy} from "./SafeWebAuthnSignerProxy.sol";
+
+/**
+ * @title Safe WebAuthn Signer Proxy
```

Comment:
fix the comments and explain this file, it's a copy of the fallback with the delegate, right?

Reply:
yes, I changes the comments